ctfo_server.h

```diff
@@ -101,15 +101,26 @@ struct CTFOHypermedia
 
 CURRENT_STRUCT(CTFOServerParams) {
   CURRENT_FIELD(api_port, int);
+  CURRENT_FIELD_DESCRIPTION(api_port, "Port to spawn CTFO API on.");
   CURRENT_FIELD(rest_port, int, 0);
+  CURRENT_FIELD_DESCRIPTION(rest_port, "Port to spawn RESTful server on (the same as api_port by default).");
   CURRENT_FIELD(midichlorians_port, int, 0);
```

Comment:
It would if I don't change the nginx config together with deploying the new version. Given that the currently running version doesn't have a config file and the new one does, I'll do my best to not forget the edit the nginx config too!